plugins/UsersManager/templates/index.twig

```diff
@@ -191,7 +196,13 @@
                         <td id="password" class="editable" ng-click='manageUsers.editUser("row{{ i|e('js') }}")'>-</td>
                         <td id="email" class="editable" ng-click='manageUsers.editUser("row{{ i|e('js') }}")'>{{ user.email }}</td>
                         <td id="alias" class="editable" ng-click='manageUsers.editUser("row{{ i|e('js') }}")'>{{ user.alias|raw }}</td>
-                        <td id="token_auth_user" class="token_auth" data-token="{{ user.token_auth }}">{{ user.token_auth|slice(0, 8) }}…</td>
+                        <td id="token_auth_user" class="token_auth" data-token="{{ user.token_auth }}">
+                            <span class="token_auth_content">{{ user.token_auth|slice(0, 8) }}…</span>
+                            <button ng-click='manageUsers.regenerateUserTokenAuth({{ user.login|json_encode }}); $event.stopPropagation();'
```

Comment:
Can we should a little popup forcing users to confirm this action? It would be similar to `div id= confirmPasswordChangeSelf ` only with a different message. And then you can show it before regenerating the token like  

```js
piwik.helper.modalConfirm('#confirmPasswordChange', {yes: function () {
                   // really regenerate it
                }});
```